### DISCUSSION TITLE
Grizzly Features

### DISCUSSION THREAD

**Discussion by PatMis16** [grafana/grizzly#202] [General]
Hi contributors to Grizzly,

I am wondering if Grizzly is still an active project or if there is something new which replaced Grizzly already. 
We need a solution to manage various ressources in Grafana like dashboards, datasources and organisations. 
Is Grizzly the right way to go?

Regards,
Patrick

**Comment by malcolmholmes** [comment]
Grizzly is an experimental tool for managing dashboards/etc. Whilst there have been many discussions around tools that *could* be built to build on the lessons from Grizzly, no discussions have yet led to anything concrete.

Rather than focusing on tools, the Grafana team is more focusing on building out schema and cleaner APIs. These are building blocks that any team will need.

Grizzly does as much as I need it to, so in that sense, it is more a stable project than an active one. However, we do still accept contributions.
